components/project-card/meta/get-involved.jsx

```diff
@@ -20,7 +20,6 @@ class GetInvolved extends React.Component {
     let props = this.props;
     let getInvolvedText = props.getInvolved ? props.getInvolved : null;
     let getInvolvedLink = props.getInvolvedUrl ? ( <a href={props.getInvolvedUrl} target="_blank" onClick={() => this.handleGetInvolvedLinkClick()}>Get Involved</a>) : null;
-
```

Comment:
Newlines after variable declarations are usually required by the linter...keep this in and I'll file a bug to make sure the linter is checking this. (Same for other new lines removed below)